core/config/config.go

```diff
@@ -61,10 +63,11 @@ var (
 func Load(ctx context.Context, db pg.DB, sdb *sinkdb.DB) (*Config, error) {
 	c := new(Config)
 	found, err := sdb.Get(ctx, "/core/config", c)
-	if err != nil {
+	if errors.Root(err) == raft.ErrUninitialized {
```

Comment:
I think maybe it might be better to handle this in cored? Like

```go
conf, err := config.Load(ctx, db, sdb)
if err != nil && errors.Root(err) != raft.ErrUninitialized {
	chainlog.Fatalkv(ctx, chainlog.KeyError, err)
}
```

and this function would stay as it was? (or maybe just have ErrUninitialized appear in its godoc)